resources/lang/en/notifications.php

```diff
@@ -122,6 +122,12 @@
                     'channel_message_compact' => ':title',
                     'channel_message_group' => 'from :username',
                 ],
+
+                'team' => [
+                    'channel_team' => ':username says ":title" on team chat',
+                    'channel_team_compact' => ':username says ":title"',
+                    'channel_team_group' => 'team chat',
+                ],
```

Comment:
In hindsight, maybe the notification grouping for channel messages should have structured differently. 🤔 
It makes more sense for team messages to have the title be `New team message` and the group display be the most recent message instead of `team chat`?